src/Components/Header/Header.js

```diff
@@ -1,13 +1,291 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./Header.scss";
 import ItemCard from "../ItemCard/ItemCard";
 
 class Header extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.myRef = React.createRef();
+    this.state = {
+      fix: false,
+      hoverAction: false,
+      subHoverAction: false,
+    };
   }
+
+  componentDidMount() {
+    window.addEventListener("scroll", this.handleScroll);
+  }
+  componentWillUnMount = () => {
+    window.removeEventListener("scroll", this.handleScroll);
+  };
+
+  handleScroll = e => {
+    const scrollTop = ("scroll", window.scrollY);
+    if (scrollTop > 160) {
+      this.setState({ fix: true });
+    } else {
+      this.setState({ fix: false });
+    }
```

Comment:
삼항연산자로 하면 더 간단해지겠네요! 
```suggestion
     this.setState({ fix: scrollTop > 160 ? true : false });
```

Reply:
오.. 생각지도못했네요 반영했습니다.